src/js/views/patients/sidebar/action/action-sidebar.scss

```diff
@@ -22,6 +22,31 @@
   }
 }
 
+.action-sidebar__activity {
+  .spinner {
+    margin-top: 100px;
+  }
+}
```

Comment:
Couldn't figure out how to do center vertical alignment. Will keep looking into it, but this is seems to work pretty well.